site/package.json

```diff
@@ -47,12 +47,13 @@
     "eslint": "^8.11.0",
     "eslint-plugin-cypress": "^2.12.1",
     "eslint-plugin-jest": "^26.1.5",
-    "jest": "^27.5.1",
+    "jest": "^28.0.3",
+    "jest-environment-jsdom": "^28.0.2",
```

Comment:
I'm not particularly familiar with the Typescript development cycle.  Is there a reason why `jest` is getting updated and `jest-environment-jsdom` is being added here?  The release notes do not seem to include any information about the change.

Reply:
After another look, `jest-environment-jsdom` is not being added here.  It was already there and the GitHub interface just seems to not show the removed line properly.  The original question about the dependencies being updated still stands though.